src/components/Pages/index.js

```diff
@@ -215,6 +232,12 @@ class VmEditPage extends React.Component {
     }
   }
 
+  componentDidUpdate () {
+    if (!this.props.isFilteredClusters && this.props.clusterSize) {
```

Comment:
Humm...might be safer to use the VM `canUserEditVm` prop here instead of looking at the clusters.

Like what was done in VmActions/index.js...

Reply:
Kept the cluster check (which really only applies to the user's ability to possibly edit any VM), but also added a specific edit check for the VM.  To get in to edit, a user will need to pass both check.